### PR TITLE
kodi: update to v19.1

### PR DESCRIPTION
**Pull request by HiassofT**

### CODE REVIEW

**Comment by HiassofT** [comment]
runtime tested on RPi4

**Comment by heitbaum** [comment]
Run tested on both Nuc11 TGL, and Odroid N2+

File: packages/mediacenter/kodi/package.mk [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="kodi"
-PKG_VERSION="1a1c5028f24a992134e2a1745519914e303eee4c"
-PKG_SHA256="b125bf10459b6d2a15f6f4740348cf8424591e1229281b12f5aeb7f8f71a5d57"
+PKG_VERSION="v19.1"
```
Comment (CvH) [inline packages/mediacenter/kodi/package.mk:6]: pls change to `19.1-Matrix` like used before

Comment (HiassofT) [reply packages/mediacenter/kodi/package.mk]: done

**Review by CvH — APPROVED** [review]

**Pull request merged**